CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [1.0.0-dev.1]
+- BREAKING: update supabase package [v1.0.0-dev.1](https://github.com/supabase-community/supabase-dart/blob/main/CHANGELOG.md#100-dev1)
```

Comment:
https://github.com/supabase-community/supabase-flutter/pull/136
https://github.com/supabase-community/supabase-flutter/pull/124
https://github.com/supabase-community/supabase-flutter/pull/131

these should be added